Flick/Cells/TrendingContentCollectionViewCell.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  TrendingContentCollectionViewCell.swift
+//  Flick
+//
+//  Created by Lucy Xu on 12/24/20.
+//  Copyright © 2020 flick. All rights reserved.
+//
+
+import UIKit
+
+class TrendingContentCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: - Private View Variables
+    private let imageView = UIImageView()
+    private let saveButton = UIButton()
+    private let shareButton = UIButton()
+    private var mediaId: Int!
+    var delegate: MediaControllerDelegate?
+    
+    static let reuseIdentifier = "TrendingContentCellReuseIdentifier"
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        backgroundColor = .clear
+        
+        imageView.layer.cornerRadius = 12
+        imageView.clipsToBounds = true
+        imageView.layer.masksToBounds = true
+        imageView.contentMode = .scaleAspectFill
+        contentView.addSubview(imageView)
+        
+        shareButton.setImage(UIImage(named: "shareButton"), for: .normal)
+        contentView.addSubview(shareButton)
+        
+        saveButton.setImage(UIImage(named: "saveButton"), for: .normal)
+        saveButton.addTarget(self, action: #selector(saveMedia), for: .touchUpInside)
+        contentView.addSubview(saveButton)
+        
+        setupConstraints()
+    }
+    
+    private func setupConstraints() {
+        
+        imageView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        
+        saveButton.snp.makeConstraints { make in
+            make.size.equalTo(CGSize(width: 36, height: 36))
+            make.trailing.equalTo(imageView).inset(12)
+            make.centerY.equalTo(imageView.snp.bottom)
+        }
+        
+        shareButton.snp.makeConstraints { make in
+            make.size.equalTo(CGSize(width: 36, height: 36))
+            make.trailing.equalTo(saveButton.snp.leading).offset(-6)
+            make.centerY.equalTo(saveButton)
+        }
+        
+    }
+    
+    @objc func saveMedia() {
+        print("saving media")
+        NetworkManager.addToMediaList(listId: 1, mediaIds: [mediaId]) { [weak self] list in
```

Comment:
so hitting the save button is saving to the 'Saved' list?

Reply:
haha ya just doing it manually to 1 for now..